src/core/src/graph_util.cpp

```diff
@@ -335,6 +336,12 @@ void serialize(const std::shared_ptr<const ov::Model>& m,
                const std::string& bin_path,
                ov::pass::Serialize::Version version) {
     ov::pass::Manager manager;
+    // TODO: if rt_info is set in python api as a string ['disable_fp16_compression_0'] = '',
```

Comment:
@jiwaszki do we have plans to export RuntimeAttribute and inherited classes to python API?
it looks like this is similar to CVS-105807

Reply:
There are no advances in Core itself. As for now Python API is blocked.

Reply:
@jiwaszki do you agree with the current workaround?

Reply:
I don't have anything against it.

CC @akuporos what do you think?